packages/jest-config/src/__tests__/normalize-test.js

```diff
@@ -76,6 +76,13 @@ describe('normalize', () => {
     }).name).toBe('custom-name');
   });
 
+  it('sets coverageReporters correctly when argv.json is set', () => {
+    expect(normalize({
+      rootDir: '/root/path/foo',
+    }, { json: true }).coverageReporters).toEqual(["json"]);
```

Comment:
please use single quotes and no spaces after `{` and before `}` in an object literal.